zabbix_metrics_watcher.py

```diff
@@ -70,7 +70,7 @@ def durations(self):
         except:
             return {}
 
-    def send_zabbix_notification(self, zabbix_host, osbs_master, concurrent_builds):
+    def send_zabbix_notification(self, zabbix_host, osbs_master, concurrent_builds, builds_in_queue):
```

Comment:
That's odd, I thought I've already pushed that to master. Right, I'll make another PR for builds in queue